addons/website_crm/views/website_crm_templates.xml

```diff
@@ -1,100 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-        <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website_form.contactus_form" customize_show="True">
+        <template id="contactus_form" name="Contact Form (Opportunity)" inherit_id="website.contactus" customize_show="True">
             <xpath expr="//span[@data-for='contactus_form']" position="before">
-                <t t-set="logged_partner" t-value="request.env['website.visitor']._get_visitor_from_request().partner_id"/>
-            </xpath>
-            <xpath expr="//span[@data-for='contactus_form']" position="attributes">
-                <attribute name="t-att-data-values">{
-                    'contact_name': request.params.get('contact_name', logged_partner.name if logged_partner else ''),
-                    'phone': request.params.get('phone', logged_partner.phone if logged_partner and logged_partner.phone else ''),
-                    'email_from': request.params.get('email_from', logged_partner.email if logged_partner else ''),
-                    'partner_name': request.params.get('partner_name', logged_partner.commercial_company_name if logged_partner else ''),
+                <t t-set="contactus_form_values" t-value="dict(contactus_form_values, **{
+                    'contact_name': request.params.get('contact_name', ''),
+                    'phone': request.params.get('phone', ''),
+                    'email_from': request.params.get('email_from', ''),
+                    'partner_name': request.params.get('partner_name', ''),
                     'name': request.params.get('name', ''),
                     'description': request.params.get('description', ''),
-                }</attribute>
+                })"/>
             </xpath>
-            <xpath expr="//div[@id='contactus_section']" position="replace">
-                <div id="contactus_section">
-                    <section class="s_website_form" data-vcss="001" data-snippet="s_website_form">
-                        <div class="container">
-                            <form id="contactus_form" action="/website_form/" method="post" enctype="multipart/form-data" data-model_name="crm.lead" data-success-mode="redirect" data-success-page="/contactus-thank-you" class="o_mark_required" data-mark="*" data-editable-form="false">
```

Comment:
@tsm-odoo  Unable to generate Lead from Contact US on Website because attribute always is `data-model_name="mail.mail". Did you miss it?

Reply:
Hello, just tested it on master and everything seems to work as expected. Which Odoo version are you using ? Could you please provide the steps to reproduce this issue ?

Reply:
@hoangtiendung070797 Did you enter edit mode, clicked on the form and changed the action option to "Create an Opportunity" ?
![image](https://user-images.githubusercontent.com/10338094/178732037-e0ce134e-6d69-4c61-827b-8837967e1320.png)

Reply:
@qsm-odoo Okay, Thank you very much. I use Odoo 15.0 and didn't see it there
cc @tsm-odoo

Reply:
@qsm-odoo does it make sense to keep this view with customize_show ? not sure...
I take a few time to understand why we got this option on the right panel, and nobody that can't read the code, could understand it imo. Maybe it could be in website module ?

HEre a good example of customer feedbacks: https://www.odoo.com/forum/website-builder-12/pass-default-values-for-form-fields-in-url-parameters-205289


//cc @pierre-pde

Reply:
Note that we have a task to re-enable the auto form target switch when installing website_crm (not only?): https://www.odoo.com/web#id=2719426&menu_id=4720&cids=1&action=327&model=project.task&view_type=form
But I couldn't come with a clean solution yet (didn't check a second time since, wasn't a priority), I made some feedback about it on this task which was created later: https://www.odoo.com/web#id=2779452&cids=1&model=project.task&view_type=form.

In a nutshell, it is not easy in python to alter the form to send lead automatically when the module is installed as there is a lot of stuff involves which in the form transformation, which is done in JS.

Reply:
It was the tradeoff that FP decided in the past.
Nothing shocking imo, just need to know that it has been changed for old user.
For a new user, it it maybe even more logic that install a module add a feature, but don't change without warning your contact us form that you have maybe customized previously...

So, imho, remove the customize_show (before v16 freeze) if none interest to be able to toggle it, because it is confusing. If you see a good reason to keep it, I'm curious to know/understand why/which purpose...

And for the rest, to debate later in a low priority task if we really want to restore previous behaviour...

Reply:
It is already removed for v16 with the frontend > backend task :+1: https://github.com/odoo/odoo/blob/4d3b26b5808320b12445735dd15dd24fc991c8e1/addons/website_crm/views/website_templates_contactus.xml#L4

About the debate, as there is no clean way to do it now anyway, that's not something we will be able to change

Reply:
@JKE-be Indeed the customize_show is just there by mistake, we'll check how to remove it in stable. For master, the customize_show="True" was removed indeed but converted as an option... also by mistake. We'll remove that with the forward-port of the stable fix.